module/Photo/Module.php

```diff
@@ -105,4 +125,24 @@ public function getServiceConfig()
         );
     }
 
+    /**
+     * Get view helper configuration.
+     *
+     * @return array
+     */
+    public function getViewHelperConfig()
+    {
+        return array(
+            'factories' => array(
+                'isAllowed' => function ($sm) {
+                    $locator = $sm->getServiceLocator();
+                    $helper = new \Application\View\Helper\Acl();
```

Comment:
Ah indeed, that was the big problem with the `isAllowed` helper.

I do have an idea to solve this. Instead of a direct `isAllowed` helper, make a general `acl` helper which takes the service locator key for the acl as argument. Then you can load a specific ACL and use it from the view as follows:

```php
$this->acl('decision_acl')->isAllowed('member', 'view');
$this->acl('photo_acl')->isAllowed('photo', 'edit');
```